src/test/java/org/opentripplanner/routing/algorithm/mapping/RaptorPathToItineraryMapperTest.java

```diff
@@ -57,6 +76,16 @@ public class RaptorPathToItineraryMapperTest {
     STOP_COSTS
   );
 
+  private static final RegularStop S1 = RegularStop
```

Comment:
There is also `TransitModelForTest.stop` or do you need a specific feed id?

Reply:
Indeed that makes the test setup simpler.